pulsar-client/src/main/java/org/apache/pulsar/client/impl/TransactionMetaStoreHandler.java

```diff
@@ -167,6 +167,8 @@ public CompletableFuture<Void> connectionOpened(ClientCnx cnx) {
                     return null;
                 });
             } else {
+                LOG.warn("Can not connect to the transaction coordinator because the protocol version {} is "
+                                + "lower than 19", cnx.getRemoteEndpointProtocolVersion());
                 registerToConnection(cnx);
```

Comment:
Just curious that if client protocol version lower than the expected version？does any txn request will throw exception？It seems the Future just completed as normal

Reply:
The broker version should keep consistent if the client proto version is lower than 18.